source/boards/share-and-collaborate.rst

```diff
@@ -7,22 +7,52 @@ Share and collaborate
+| View                        |   X       |    X       |     X         |    X       |
++-----------------------------+-----------+------------+---------------+------------+
+
 The level of access to a board is determined by a user’s assigned board role.
 
-- **Admin:** Can modify the board, its contents, and its permissions. By default, board creators are also admins of the board.
-- **Editor:** Can modify the board and its contents.
+- **Admin**: Can modify the board, its contents, and its permissions. By default, board creators are also admins of the board.
+- **Editor**: Can modify the board and its contents.
+- **Commenter**: Can add comments to cards.
+- **Viewer**: Can view the board and its contents but can't comment or edit the board.
 
 Manage team access
 ~~~~~~~~~~~~~~~~~~~
 
-Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a default board role for everyone on the team. 
+Board admins can manage team access to their board by selecting **Share** in the top-right corner of the board. On the dropdown next to **Everyone at… Team** option, select a minimum board role for everyone on the team. You can also easily assign the new roles to the entire team and/or to individual team members.
+
+Minimum default board roles reduce permission ambiguity and prevent security loopholes. Without a minimum role, there are scenarios where individual board members can elevate their own permissions without knowledge of the admin. For example, if an individual board member is assigned the **Viewer** role, but the default team role is set to **Editor**, then the individual can elevate their own permissions by leaving the board and rejoining the board. They would then be assigned the **Editor** role. 
 
-Depending on the role selected, everyone on the team will have access to the board with the permissions from the role selected. Users can get elevated permissions based on their individual board membership. The default team access for a newly created board is **None**, which means nobody on the team has access to the board.
+The minimum default role means that board admins can't assign individual board members a role lower than the team role. If the team role is set to **Editor** then the board admin will only be able to assign the **Editor** or **Admin** role to individual team members. Lower roles will not be available for selection unless the admin changes the minimum board role.
+
+Depending on the role selected, everyone on the team will have access to the board with a minumum of the permissions from the role selected. Users can get elevated permissions based on their individual board membership. The default team access for a newly created board is **None**, which means nobody on the team has access to the board.
 
 Manage individual board membership
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
@justinegeffen Can we add the autocomplete content to the end of the `Manage individual board membership` section? Last 2 paragraphs are new:

>Only board admins can manage user permissions on a board, including adding, changing, and removing members.
>
>To add individual users from the team as explicit members of the board, open the Share dialog on the board, search for individual team members, then assign a role to set their permissions for the board. The role for individual board members overrides any role specified for team access.
>
>To change a board member’s role, open the Share dialog, select the role dropdown next to the user’s name, then select another role from the list.
>
>To remove a member from a board, open the Share dialog, select the role dropdown next to the user’s name, then select Remove member.
>
>Board admins can also add individual members using the autocomplete list from @mentions and the person properties. To add an individual from the autocomplete list, type their username in an @mention or in the **Person** or **Multi-person** properties, then assign a role to the user from the confirmation dialog, and select **Add to board**. 
>
>On boards with [team access](https://docs.mattermost.com/boards/share-and-collaborate.html#manage-team-access), board members with **Editor** or **Commenter** roles can also add individuals to the board from the autocomplete list. Board members added in this manner will be assigned the default minimum board role.